lib/Contributors/index.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Icon, Button, AvatarWithInformation, SearchDropdown } from '../';
+
+class Contributors extends Component {
+  static propTypes = {
+    currentAssignees: PropTypes.arrayOf(PropTypes.shape()),
+    currentViewers: PropTypes.arrayOf(PropTypes.shape()),
+    searchResults: PropTypes.arrayOf(PropTypes.shape()),
+    removeAssignee: PropTypes.func.isRequired,
+    handleSearchChange: PropTypes.func.isRequired,
+    handleClearResults: PropTypes.func.isRequired,
+    userCanAddAssignee: PropTypes.bool,
+    emptyText: PropTypes.string
+  };
+
+  static defaultProps = {
+    currentAssignees: [],
+    currentViewers: [],
+    searchResults: [],
+    emptyText: 'There is no one assigned yet',
+    userCanAddAssignee: false
+  };
+  constructor() {
+    super();
+    this.state = {
+      showSearch: false
+    };
+  }
+
+  getCurrentViewers = () =>
+    this.props.currentViewers.map(viewer => (
+      <AvatarWithInformation
+        key={viewer.id}
+        className="viewing-list-item"
+        url={viewer.avatar}
+        name={viewer.name}
+        initials={viewer.initials}
+        colour={viewer.colour}
+        smallSize
+      />
+    ));
+
+  getcurrentAssignees = () => {
+    const assignees = this.props.currentAssignees;
```

Comment:
Maybe just deconstruct `currentAssignees` and map over that? Feels strange how we're assigning a new const here calling it `assignees` when we already have `currentAssignees`. I'm easy though, let me know if you disagree 👍